.github/CONTRIBUTING.md

```diff
@@ -178,6 +179,58 @@ Additionally we will reference any new contributors between the release versions
     @dark0dave made their first contribution in #1
 ```
 
+## Releases
+
+We use [semantic versioning](https://semver.org/) in kubent.
+
+> Given a version number MAJOR.MINOR.PATCH, increment the:
+>
+> - MAJOR version when you make incompatible API changes
+> - MINOR version when you add functionality in a backward compatible manner
+> - PATCH version when you make backward compatible bug fixes
+>
+> Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.
+
+### Triggering a release
+
+Releases are triggered by pushing a new tag, it is **imperative** you keep to the style of tag defined in the cliff.toml file otherwise git-cliff will not generate valid release notes.
```

Comment:
Maybe we should be clear this is done manually?

```suggestion
Releases are triggered by pushing a new tag (which is done manually), it is **imperative** you keep to the style of tag defined in the cliff.toml file otherwise git-cliff will not generate valid release notes.
```

Reply:
I hear what you are saying, I feel it adds confusoin as the release section is automatic. But tagging works as described in the git docs. As a compromise, would you like me to add a link to git tag documentation?

https://git-scm.com/book/en/v2/Git-Basics-Tagging

Reply:
Hi @dark0dave, actually what I meant is that we have an automation for the nightly tags as `x.y.z-<hash>` in nightly CI [here](https://github.com/doitintl/kube-no-trouble/blob/master/.github/workflows/main.yaml), however I couldn't see a tag creation as semantic tag `x.y.z`, please correct me if I'm wrong. I assume we run the tag command manually in our terminal and push it. Then we will have the release created as a draft as we can see  [here](https://github.com/doitintl/kube-no-trouble/blob/2f8b617d3dda6ffdabab146c00addd222a3458cd/.github/workflows/main.yaml#L176C82-L176C85)

Reply:
Yes your correct! 

> Releases are triggered by pushing a new tag,

I thought this covered it, as the act of pushing and creating a tag triggers the release.

I the work flow is 

```sh
$ git tag 0.7.4
$ git push 0.7.4
```
From here the release + release notes are created automatically.

I can clarify that if thats what you want?

Reply:
Thanks for confirming my assumptions! As non native English speaker I wasn't sure keeping this implicit was clear enough.

But re-reading again I believe this is fine.

Fair enough, happy to get this merged, thanks for all effort on this! 💪